compiler/cpp/src/generate/t_rb_generator.cc

```diff
@@ -706,7 +706,7 @@ void t_rb_generator::generate_field_defns(t_rb_ofstream& out, t_struct* tstruct)
     // generate the field docstrings within the FIELDS constant. no real better place...
     generate_rdoc(out, *f_iter);
 
-    out.indent() << upcase_string((*f_iter)->get_name()) << " => ";
+    out.indent() << "THRIFT_FIELD_INDEX_" << upcase_string((*f_iter)->get_name()) << " => ";
```

Comment:
just to understand what is the `THRIFT_FIELD_INDEX_` ?

Reply:
Given the fact we now have `NAME` and `NAMESPACE` defined in those structs for the reflection system. We had some conflict in  the argument name if a struct field was `name` or `namespace` hence,  i decided to make field index constant `THRIFT_FIELD_INDEX_NAME` rather than `NAME`